grype/presenter/models/match.go

```diff
@@ -60,30 +60,36 @@ func newMatch(m match.Match, p pkg.Package, metadataProvider vulnerability.Metad
 	}, nil
 }
 
-var _ sort.Interface = (*ByName)(nil)
+var _ sort.Interface = (*MatchSort)(nil)
```

Comment:
Renamed since there is more than just `ByName` being sorted here. This is now the default match sort. We can always enhance this by adding other sort methods that a user could influence via flag or config option